### ISSUE TITLE
[Suggestion] Add more gears

### ISSUE THREAD

**Issue by 016Nojr** [CoFH/Feedback#1235]
Since Thermal is adding gears, plates, and everything making a big amount of modpacks use Thermal Foundation materials as default, but one thing I am missing to have a "complete pack" with only Thermal Foundation materials is a Wooden Gear, Stone Gear and a Diamond Gear. Now those gears are coming from mods where the gear texture is completely different from Thermal's texturing.

I guess it hasnt been added because you dont have a plan to use it for anything, but maybe you will when you add them?

In short, I would appreciate if you could add these gears (and oredict them):

- Wooden Gear
- Stone Gear
- Diamond Gear

**Comment by KingLemming** [comment]
Sure, okay.

**Comment by KingLemming** [comment]
And Emerald.

**Comment by 016Nojr** [comment]
Sweet, thanks! :)
